src/main/java/com/fasterxml/uuid/impl/TimeBasedEpochGenerator.java

```diff
@@ -69,25 +76,34 @@ public TimeBasedEpochGenerator(Random rnd)
     /* UUID generation
     /**********************************************************************
      */
-
+    
     @Override
     public UUID generate()
     {
-        final long rawTimestamp = System.currentTimeMillis();
-        final byte[] rnd = new byte[10];
-        _random.nextBytes(rnd);
-
-        // Use only 48 lowest bits as per spec, next 16 bit from random
```

Comment:
Could you add a modified comment still mentioning the logic on augmenting 48 bits of clock with sequence generated using entropy as base? (or something along those lines)